src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.html

```diff
@@ -72,7 +72,7 @@
       <p class="col-text">{{ 'ubs-tariffs.city' | translate }}</p>
       <div class="input-wrapper">
         <input
-          placeholder="{{ 'ubs-tariffs.placeholder-locality' | translate }}"
```

Comment:
Can't you use it just like this? 
`placeholder="{{ selectedCities.length }} {{ 'ubs-tariffs.placeholder-locality' | translate }}"`

Reply:
Try this way:
`placeholder="{{ selectedCities.length ? selectedCities.length + ' ' + ('ubs-tariffs.selected' | translate) : ('ubs-tariffs.placeholder-locality' | translate)  }}"`

Reply:
Yes, this works correctly.
However, I guess that it may be too complicated to read

Reply:
I think it's better to put it in template since it's displaying logic. It should look ok if broken into 3 lines.